CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ TBD
 - Added auto refresh checkbox to requests page
 - Added support for cross-server parent/child requests
 - Reuse existing rabbit admin queue by purging messages instead of creating new queue
+- Refactored Mongo pruner to use APScheduler
```

Comment:
Since we cut the release, need to move this to a new version. Then it is good to go